app/src/main/res/xml/grayscale_icon_map.xml

```diff
@@ -2,119 +2,76 @@
 
 <icons>
   <icon drawable="@drawable/aliucord_installer" package="com.aliucord.manager" name="Aliucord Manager"/>
-  <icon drawable="@drawable/androtainer" package="com.dokeraj.androtainer" name="AndroTainer"/>
   <icon drawable="@drawable/animiru" package="xyz.Quickdev.Animiru.mi" name="Animiru"/>
   <icon drawable="@drawable/bandbond" package="se.droid.bandbond" name="Bandbond"/>
-  <icon drawable="@drawable/bergfex" package="com.bergfex.mobile.android" name="bergfex"/>
   <icon drawable="@drawable/bitrix24" package="com.bitrix24.android" name="Bitrix24"/>
-  <icon drawable="@drawable/blindmate" package="eu.appiphany.blindmate" name="blindmate"/>
   <icon drawable="@drawable/camera" package="com.google.android.GoogleCamera.CamFour" name="Camera"/>
   <icon drawable="@drawable/camera_date_folders" package="de.kromke.andreas.cameradatefolders" name="Camera Date Folders"/>
   <icon drawable="@drawable/coinvero" package="com.candlestick_app" name="Coinvero"/>
   <icon drawable="@drawable/combined" package="de.peterge.combined" name="Combined!"/>
-  <icon drawable="@drawable/cookbook" package="de.uni_potsdam.hpi.openmensa" name="OpenMensa"/>
```

Comment:
No svg present so removed it

Reply:
`cookbook.svg` was renamed to `generic_silverware`. It can be added back with the other drawable

Reply:
Add generic_silverware insted of coobbook drawbles and moved to appfilter

https://github.com/LawnchairLauncher/lawnicons/pull/1014/commits/9e6629da3297d1edbcb0f65319e7e10ecb14ddac#diff-78a61c73b85c0c3925b77023f7eb3e58598e746095876a601b21c72023c9a846R651